config/intellij-idea-inspections.xml

```diff
@@ -2134,6 +2136,16 @@
                 <option value="ClassLoaderInstantiation" />
                 <!-- There are asserts in DetailASTTest#checkNode, but idea does not see them -->
                 <option value="JUnitTestMethodWithNoAssertions" />
+                <!-- we use it in Main class -->
+                <option value="CallToSystemExit" />
+                <!-- we use it for testing purposes -->
+                <option value="unused" />
+                <!-- we use it for testing purposes -->
+                <option value="UseOfPropertiesAsHashtable" />
+                <!-- till #4861, #4862, #4863, #4864, #4866 -->
+                <option value="ThisEscapedInObjectConstruction" />
+                <!-- generates false-positive in CyclomaticComplexityCheck -->
+                <option value="MismatchedQueryAndUpdateOfCollection" />
```

Comment:
why you allow `MismatchedQueryAndUpdateOfCollection` to suppress , and do not use this in this PR.
If not required now - it is not required.